domain/externalcontroller/state/state.go

```diff
@@ -129,9 +129,9 @@ VALUES (?, ?, ?)
 			q := `
 INSERT INTO external_model (uuid, controller_uuid)
 VALUES (?, ?)
-  ON CONFLICT(uuid) DO UPDATE SET controller_uuid=?`[1:]
+  ON CONFLICT(uuid) DO UPDATE SET controller_uuid=excluded.controller_uuid`[1:]
```

Comment:
Didn't know about `excluded`, that makes things a little tidier.

Reply:
Yeah hopefully minimize a small set of logic bugs that we could encounter.